script.js

```diff
@@ -1,44 +1,77 @@
-const items = [
-  {
-    id: 0,
-    nome: "gastly",
-    img: "gengar.jpg",
-    quantidade: 0,
-  },
-  {
-    id: 1,
-    nome: "haunter",
-    img: "gengar.jpg",
-    quantidade: 0,
-  },
-  {
-    id: 2,
-    nome: "gengar",
-    img: "gengar.jpg",
-    quantidade: 0,
-  },
-];
-
-inicializarloja = () => {
-  var containerProdutos = document.getElementById("produtos");
-  items.map((val) => {
-    //TODO : refatorar para usar template literals
-    //https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals
-    containerProdutos.innerHTML +=
-      `
-                <div class= "produtos-single">
-                <img src="` +
-      val.img +
-      `" />
-                <p>` +
-      val.nome +
-      `</p>
-                <a href="` +
-      val.id +
-      `">Adicionar ao carrinho<a/>
-                <div/>
-            `;
-  });
-};
-
-window.onload = inicializarloja;
+const items = [
+  {
+    id: 0,
+    nome: "gastly",
+    img: "https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/92.png",

```

Comment:
Além disso, veja que essa URL base vai ser repetida em todo o array de items.
Considere pelo menos mover `https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/` para uma variável externa (como `const BASE_IMAGE_URL = "`https://raw.githubusercontent.com/PokeAPI/sprites/master/sprites/pokemon/other/official-artwork/"`)

```suggestion
    img: BASE_IMAGE_URL+"92.png",

```

Spoiler: você não precisa no futuro dessa entrada img nos objetos dos items. Consegue ver o porquê?